.github/CONTRIBUTING.md

```diff
@@ -44,15 +44,23 @@ git clone git@github.com:bpmn-io/diagram-js.git
 
 git clone git@github.com:bpmn-io/bpmn-js.git
 (cd bpmn-js && npm install && npm link ../diagram-js)
+```
+
+For details consult our in depth [setup guide](https://github.com/bpmn-io/bpmn-js/blob/master/docs/project/SETUP.md).
+
 
-// Run the test suite
-npm test
+### Build and Run the Project
 
-// Running the test suite with every file change
+```sh
```

Comment:
(Minor) I would put these three statements in three different blocks and add on top "You have three options to build and run the project". IMO would avoid any confusion to run these three statements one after the other